drivers/copy/copy_linux.go

```diff
@@ -1,4 +1,4 @@
-// +build linux
+// +build cgo
```

Comment:
Looking at the `#include <linux/fs.h>` below, this still should be Linux-only.

Reply:
The `linux` build constraint is implicit based on the file's name.

Reply:
Ah, I thought that the explicit comment overrides the tag implied by the file name. Thanks.